docs/i18n-languages/vietnamese/README.md

```diff
@@ -0,0 +1,158 @@
+![freeCodeCamp.org Social Banner](https://s3.amazonaws.com/freecodecamp/wide-social-banner.png)
+
+[![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat)](http://makeapullrequest.com)
+[![first-timers-only Friendly](https://img.shields.io/badge/first--timers--only-friendly-blue.svg)](http://www.firsttimersonly.com/)
+[![Open Source Helpers](https://www.codetriage.com/freecodecamp/freecodecamp/badges/users.svg)](https://www.codetriage.com/freecodecamp/freecodecamp)
+[![Setup Automated](https://img.shields.io/badge/setup-automated-blue?logo=gitpod)](https://gitpod.io/from-referrer/)
+
+## Mã nguồn mở và chương trình học của freeCodeCamp.org
+
+[freeCodeCamp.org](https:/www.freecodecamp.org) là một giao diện thân thiện cho bạn học lập trình miễn phí. Website được tạo ra nhờ sự đóng góp của [donor-supported 501(c)(3) nonprofit](https://donate.freecodecamp.org) với mục đích để giúp hàng triệu người tiếp cận với chuyên nghành công nghệ thông tin. Cộng đồng của chúng tôi đã giùp hơn 10,000 người có được công việc lập trình đầu tiên của mình.
+
+Chương trình học lập trình web full-stack hoàn toàn miễn phí và tự học theo thời gian của bạn. Chúng tôi có hơn hàng ngàn câu hỏi lập trình tự tương tác để giúp bạn tiến triễn kỹ năng lập trình của mình.
+
+## Mục lục
+
+- [Các chứng Chỉ](#chứng-chỉ)
+- [Nền tản học tập](#nền-tảng-học-tập)
+- [Báo lỗi và vấn đề](#báo-lỗi-và-vấn-đề)
+- [Báo cáo vấn đề bảo mật](#Báo-cáo-vấn-đề-bảo-mật)
+- [Đóng góp](#đóng-góp)
+- [Tình trạng nền tảng, xây dựng và triển khai](#tình-trạng-nền-tảng,-xây-dựng-và-triển-khai)
+- [Giấy phép](#giấy-phép)
```

Comment:
These table of contents doesn't contain any links as it is in the official README.md

Reply:
@nguyenami  Please translate the latest version of the [English README.md](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/README.md) file on the `master` branch.  This may mean needing to remove obsolete content and add new content.

Reply:
I updated the table of content, it should have the same as the official README.md.
I translated directly from the README.md on the master branch so it should have everything on it.

```
- [Các chứng chỉ](#certifications)
- [Nền tản học tập](#the-learning-platforms)
- [Báo lỗi và vấn đề](#found-a-bug)
- [Báo cáo vấn đề bảo mật](#found-a-security-issue)
- [Đóng góp](#contributing)
- [Tình trạng nền tảng, xây dựng và triển khai](#platform-build-and-deployment-status)
- [Giấy phép](#license)
```